tests/conf.js

```diff
@@ -1,5 +1,7 @@
 const utils = require('./utils');
 const chai = require('chai');
+const environment = require('./utils/environment');
+const browserUtils = require('./utils/browser');
```

Comment:
I'd like to move away from a `utils` directory. I'd like to just have a "domain", where things just have their place. "utils" is just begging to be a random dumping-ground. From here the next obvious question becomes, where do we put stuff then? We just kind of need a place that represents "all the stuff". We'll have lots of files and directories in there, but that base level directory still needs a name. I think `domain` works well, but am open to most other options as long as they aren't just synonyms for "utils" :)

Reply:
Yeah I think domain works for now.